app/access/migrations/0001_initial.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 5.0.9 on 2024-10-16 14:15
+
+import django.db.models.deletion
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('provider', '0006_alter_provider_acronym_it_alter_provider_acronym_rm_and_more'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                (
+                    'username',
+                    models.CharField(primary_key=True, serialize=False, verbose_name='User name')
```

Comment:
Why use a `primary_key` rather than the default `AutoField` and `unique`?

Reply:
Not sure what you mean. Do you suggest having two fields, one a [numerical `AutoField`](https://docs.djangoproject.com/en/5.1/ref/models/fields/#autofield) with an ID given by Django and additionally a `CharField` with `unique=True`? I don't see a reason to have both a numerical ID and a textual ID... can you explain?

Reply:
I think it is in general a bad idea to mix primary keys with information, since they will be used for foreign keys and if you want to change them (say in this case you have a typing error in the username you want fix), you need cascading. I've made very bad experience using this approach.

Reply:
Good point, makes total sense.

Will adapt it to have two fields:
- An implicit `id` of type `BigAutoField`
- A `username` of type `CharField(unique=True)`